sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -347,6 +347,72 @@ export interface KeyVaultCertificateWithPolicy extends KeyVaultCertificate {
     readonly policy?: CertificatePolicy;
 }
 
+// @public
+export const enum KnownApiVersion72Preview {
+    Seven2Preview = "7.2-preview"
+}
+
+// @public
+export const enum KnownDeletionRecoveryLevel {
+    CustomizedRecoverable = "CustomizedRecoverable",
+    CustomizedRecoverableProtectedSubscription = "CustomizedRecoverable+ProtectedSubscription",
+    CustomizedRecoverablePurgeable = "CustomizedRecoverable+Purgeable",
+    Purgeable = "Purgeable",
+    Recoverable = "Recoverable",
+    RecoverableProtectedSubscription = "Recoverable+ProtectedSubscription",
+    RecoverablePurgeable = "Recoverable+Purgeable"
+}
+
+// @public
+export const enum KnownJsonWebKeyCurveName {
+    // (undocumented)
+    P256 = "P-256",
+    // (undocumented)
+    P256K = "P-256K",
+    // (undocumented)
+    P384 = "P-384",
+    // (undocumented)
+    P521 = "P-521"
+}
+
+// @public
+export const enum KnownJsonWebKeyType {
+    // (undocumented)
+    EC = "EC",
+    // (undocumented)
+    ECHSM = "EC-HSM",
+    // (undocumented)
+    Oct = "oct",
+    // (undocumented)
+    OctHSM = "oct-HSM",
+    // (undocumented)
+    RSA = "RSA",
+    // (undocumented)
+    RSAHSM = "RSA-HSM"
+}
+
+// @public
+export const enum KnownKeyUsageType {
+    // (undocumented)
```

Comment:
Please document these values.

Reply:
These are simple enum values that are part of the generated code which comes from the swagger. So, It should be ok even if it is not documented.